src/js/layer.js

```diff
@@ -249,39 +244,36 @@ define([
                     width: 1,
                 },
             },
-            visualVariables: [
-                {
-                    type: "size",
-                    field: "Confirmed",
-                    stops: [
-                        {
-                            value: 0,
-                            size: 5,
-                            label: "<15",
-                        },
-                        {
-                            value: 15,
-                            size: 20,
-                            label: "<30",
-                        },
-                        {
-                            value: 30,
-                            size: 30,
-                            label: "<100",
-                        },
-                        {
-                            value: 100,
-                            size: 45,
-                            label: "<500",
-                        },
-                        {
-                            value: 500,
-                            size: 75,
-                            label: "500+",
-                        },
-                    ],
-                },
-            ],
+            visualVariables: [{
+                type: "size",
+                field: "Confirmed",
+                stops: [{
+                        //     value: 0,
+                        //     size: 5,
+                        //     label: "<15",
+                        // },
+                        // {
+                        value: 24,
+                        size: 15,
+                        label: "less than 25",
+                    },
+                    {
+                        value: 99,
+                        size: 30,
+                        label: "less than 100",
+                    },
+                    {
+                        value: 499,
```

Comment:
```suggestion
                        value: 250,
```

Making this value 499 makes it so this stop never gets hit (Unless the value is 499-500)